scripts/pre_commit_linter.py

```diff
@@ -666,15 +668,53 @@ def _check_import_order(all_files):
 
     return summary_messages
 
+
+def _check_def_spacing(all_files):
+    """This function is used to check the number of blank lines
+    left above each class, function and method defintion.
+    """
+    print 'Starting def-spacing checks'
+    print '----------------------------------------'
+    print ''
+    pycodestyle_config_path = os.path.join(os.getcwd(), 'tox.ini')
+    summary_messages = []
+    # Selecting only Python files to check for errors
+    all_files = [
+        filename for filename in all_files if not
+        any(fnmatch.fnmatch(filename, pattern) for pattern in EXCLUDED_PATHS)
+        and filename.endswith('.py')]
+    failed = False
+    style_guide = pycodestyle.StyleGuide(config_file=pycodestyle_config_path)
+    report = style_guide.check_files(all_files)
+    report.print_statistics()
+    print '----------------------------------------'
+    print ''
+    if report.get_count() != 0:
+        failed = True
+    if failed:
```

Comment:
Here you can simply do a condition check on `report.get_count() != 0`

Reply:
Okay, got it!